File: gluecodium/src/test/resources/smoke/skip/output/android/com/example/smoke/InheritFromSkippedImpl.java
/*
 *
 */
package com.example.smoke;
import com.example.NativeBase;
class InheritFromSkippedImpl extends NativeBase implements InheritFromSkipped {
    protected InheritFromSkippedImpl(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }
    private static native void disposeNativeHandle(long nativeHandle);
    @Override
    public native boolean notInSwift(final boolean input);
    @Override
    public native float notInDart(final float input);
    @Override
    public native boolean isSkippedInSwift();
    @Override
    public native void setSkippedInSwift(final boolean value);
    @Override
    public native float getSkippedInDart();
    @Override
    public native void setSkippedInDart(final float value);
}

File: gluecodium/src/test/resources/smoke/skip/output/android-kotlin/com/example/smoke/InheritFromSkippedImpl.kt
/*

 *
 */

@file:JvmName("InheritFromSkippedImpl")

package com.example.smoke

import com.example.NativeBase

/**
 * @suppress
 */
class InheritFromSkippedImpl : NativeBase, InheritFromSkipped {
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}





    override external fun notInJava(input: String) : String


    override external fun notInSwift(input: Boolean) : Boolean


    override external fun notInDart(input: Float) : Float
    override var skippedInJava: String
        external get
        external set

    override var isSkippedInSwift: Boolean
        external get
        external set

    override var skippedInDart: Float
        external get
        external set


    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
    }
}
